distros/ubuntu1604/control

```diff
@@ -76,6 +79,9 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}
         ,rsyslog | system-log-daemon
         ,zip
         ,libpcre3
+        ,libssl
```

Comment:
Not sure this is right for all versions of Ubuntu. It might need to be 'libssl|libssl1.0.0' or 'libssl|libssl1.1' . See: https://forums.zoneminder.com/viewtopic.php?f=39&t=28294

Reply:
Correct, just pulled ubuntu 12 and 16 dockers. `libssl1.0.0` works on both. committed.

Reply:
https://github.com/ZoneMinder/zoneminder/pull/2598/commits/33db7e1e35944668f3a5b9db9b424d6f4fd2967a